src/stim/cmd/command_convert.test.cc

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+
+#include "stim/main_namespaced.test.h"
+#include "stim/test_util.test.h"
+
+using namespace stim;
+
+class ConvertTest
+    : public testing::TestWithParam<
+          std::tuple<std::string, std::tuple<std::string, std::string>, std::tuple<std::string, std::string>>> {
+   protected:
+    void SetUp() override {
+        tmp.write_contents(R"CIRCUIT(
```

Comment:
Don't test just one circuit. Use different circuits. Don't share circuits across all the tests like this.

Reply:
done